packages/docs/locales/en/en.json

```diff
@@ -293,6 +294,21 @@
         "outlined": "Toggles shadow of `va-card`"
       }
     },
+    "VaImage": {
+      "props": {
+        "ratio": "Width to height ratio.",
+        "contain": "Background size. If false, results to `cover`"
```

Comment:
Doesn't tell me much.
In fact, obfuscates, that `background-size`  is a css property.
Imagine user doesn't know what background-size means by heart and just wants to solve some particular problem.